Source/BestProjectEver/Public/Character/BPE_BaseCharacter.h

```diff
@@ -63,14 +63,17 @@ class BESTPROJECTEVER_API ABPE_BaseCharacter : public ACharacter
 	UFUNCTION()
 	void HandleCharacterDeath();
 
+	UFUNCTION()
+	void HandleCharacterDamage(UBPE_HealthComponent* CurrentHealthComponent, float CurrentHealth, float MaxHealth);
+
 	//------------------------------------------------------------------------------------------------------------------
 	// Sounds And Effects
 
 	/** play sound cue */
 	virtual void PlaySound(USoundCue* Sound);
 
 	/** play anim montage */
-	virtual void PlayMontage(UAnimMontage* Montage, float PlayRate = 1.0f);
+	virtual void PlayMontage(UAnimMontage* Montage, FName SectionName, float PlayRate = 1.0f);
```

Comment:
maybe let the params, except PlayRate, be const?

Reply:
![image](https://user-images.githubusercontent.com/85327912/173151551-f59648bb-0ad9-4d06-bbb1-63f0b1ef9c87.png)

Reply:
I can not let montage const because Montage play receives a non const ptr